misk/src/main/kotlin/misk/security/authz/AccessInterceptor.kt

```diff
@@ -39,42 +39,63 @@ internal class AccessInterceptor private constructor(
 
   internal class Factory @Inject internal constructor(
     private val caller: @JvmSuppressWildcards ActionScoped<MiskCaller?>,
-    @Suppress("PLATFORM_CLASS_MAPPED_TO_KOTLIN") private val accessAnnotations: java.util.List<out AccessAnnotation>
+    private val accessAnnotationEntries: List<AccessAnnotationEntry>
   ) : ApplicationInterceptor.Factory {
     override fun create(action: Action): ApplicationInterceptor? {
       // Gather all of the access annotations on this action.
-      val actionAnnotations = mutableListOf<AccessAnnotation>()
-      val authenticated = action.function.findAnnotation<Authenticated>()
-      if (authenticated != null) {
-        actionAnnotations += authenticated.toAccessAnnotation()
+      val actionAccessAnnotations = mutableListOf<AccessAnnotationEntry>()
```

Comment:
i find this var name vs. `accessAnnotationEntries` confusing, since they are both sets of `AccessAnnotationEntry`. what is the diff? can we name them better?